src/index.html

```diff
@@ -1,49 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="styles/main.css">
-  <link rel="stylesheet" href="styles/footer.css">
-  <link rel='icon' href='img/favicon.ico' type='image/x-icon'/ >
-</head>
-<body>
-  <div class="collabarators">Known collabarators:
-    <ul>
-        <li>Laurence Fishburne - The Matrix, John Wick</li>
-        <li>Sandra Bullock - Speed</li>
-        <li>Gary Oldman - Bram Stoker's Dracula</li>
-        <li>Al Pacino - The Devil's Advocate</li>
-        <li>Alex Winter - Bill and Ted's Excellent Adventure</li>
-        <li>Robert Mailhouse - Dogstar (band)</li>
-    </ul>
-  </div>
-  <div class="name">Keanu Reeves</div>
-  <img src="img/Keanu_Reeves.jpg" alt="A picture of Keanu Reeves" width="400px">
-  <div class="aliases">
-    <ul>
-        <li>Norman Kreeves</li>
-        <li>K.C. Reeves</li>
-        <li>Keannu Reeves</li>
-        <li>Chuck Spadina</li>
-        <li>Neo</li>
-    </ul>
-  </div>
-  <div>Country of residence: United States</div>
-
-  <footer>
-
-    <div class="container">
-
-      <a class="icon" href="https://twitter.com/keanuthings"><img src="img/icon-twitter.svg"></a>
-      <a class="icon" href="https://www.facebook.com/pages/category/Actor/Keanu-Reeves-Official-395800254521088/"><img src="img/icon-facebook.svg"></a>
-      <a class="icon" href="https://www.instagram.com/explore/tags/keanureeves/?hl=en"><img src="img/icon-instagram.svg"></a>
-
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/main.css" />
+    <link rel="stylesheet" href="styles/executive.css" />
+    <link rel="stylesheet" href="styles/footer.css" />
+    <link rel='icon' href='img/favicon.ico' type='image/x-icon'/ >
+  </head>
+  <body>
+    <div class="content">
```

Comment:
Should this be "container" instead of "content"? Not sure if you were going for the 1000px rule we talked about here.